jobs/business-sync/devops/gcp/clouddeploy.yaml

```diff
@@ -59,19 +59,6 @@ serialPipeline:
       cloudsql-instances: "a083gt-test:northamerica-northeast1:businesses-db-test"
       run-command: "./run.sh"
       service-account: "sa-job@a083gt-test.iam.gserviceaccount.com"
- - targetId: a083gt-sandbox
-   profiles: [sandbox]
-   strategy:
-    standard:
-      verify: false
-   deployParameters:
-   - values:
-      deploy-env: "sandbox"
```

Comment:
there is sandbox env in gcp, just no sandbox env in config.py. you can leave this, just replace deploy-env: "sandbox" with deploy-env: "production"